torch/distributions.py

```diff
@@ -65,7 +66,8 @@ def sample_n(self, n):
         Generates n samples or n batches of samples if the distribution parameters
         are batched.
         """
-        raise NotImplementedError
+        warnings.warn('sample_n will be deprecated. Use .sample(n) instead', PendingDeprecationWarning)
```

Comment:
n will be need to wrapped in a tuple. The code that this warning suggests doesn’t work